.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["next", "prettier"]
```

Comment:
ESLint is a tool to allow us to lint our codebase. Luckily, Next.js provides us with a configuration out of the box!